engine/src/main/java/org/terasology/rendering/assets/shader/ShaderProgramFeature.java

```diff
@@ -21,11 +21,11 @@
 /**
  */
 public enum ShaderProgramFeature {
-    FEATURE_REFRACTIVE_PASS (0b00000001),
-    FEATURE_ALPHA_REJECT (0b00000010),
-    FEATURE_LIGHT_POINT (0b00000100),
-    FEATURE_LIGHT_DIRECTIONAL (0b00001000),
-    FEATURE_USE_MATRIX_STACK (0b00100000),
+    FEATURE_REFRACTIVE_PASS      (0b00000001),
+    FEATURE_ALPHA_REJECT         (0b00000010),
+    FEATURE_LIGHT_POINT          (0b00000100),
+    FEATURE_LIGHT_DIRECTIONAL    (0b00001000),
+    FEATURE_USE_MATRIX_STACK     (0b00100000),
```

Comment:
@LadySerenaKitty: I know, and I personally agree it's clearer. But I'd like @Cervator's greenlight on this as we did discuss the issue of formatting spaces and he mentioned in general they aren't a good idea. Perhaps this is a good exception. Let's just wait and see what he says.